.circleci/config.yml

```
@@ -178,8 +190,8 @@ jobs:
 
             nyprsetuptools DockerDeploy \
               --fargate \
-              --cpu=256 \
-              --memory-reservation=512 \
+              --cpu=$CPU \
+              --memory-reservation=$MEMORY/ \
```

why the `/`?

---

failed deploy, seems like just a typo

```suggestion
              --memory-reservation=$MEMORY \
```